src/guides/v2.4/comp-mgr/dev_add-update.md

```diff
@@ -0,0 +1,79 @@
+---
+group: software-update-guide
+title: Add or update components
+menu_title: Add or update components
+menu_order: 5
+menu_node:
+functional_areas:
+  - Upgrade
+redirect_from: /guides/v.24/install-gde/install/cli/dev_add-update.html
```

Comment:
This format looks incorrect...at least it is different than the previous one.

Reply:
What looks incorrect about it?

Reply:
v.24 should be v2.4